src/stories/icon/icon-heart/IconHeart.tsx

```diff
@@ -0,0 +1,21 @@
+export type IconHeartProps = {
+  fill?: boolean;
+};
+
+export const IconHeart = ({ fill }: IconHeartProps) => {
+  return (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill={fill ? "#000" : "none"}
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M11.5 20L10.2675 18.921C5.89 15.1035 3 12.5858 3 9.49591C3 6.9782 5.057 5 7.675 5C9.154 5 10.5735 5.66213 11.5 6.70845C12.4265 5.66213 13.846 5 15.325 5C17.943 5 20 6.9782 20 9.49591C20 12.5858 17.11 15.1035 12.7325 18.9292L11.5 20Z"
```

Comment:
As far as I can tell this is copy/pasted from `public/icons/basic/icon-heart.svg`. Please add a reference to this as a code comment referencing this to help future developers understand why this might suddenly deviate if the "official" icon is updated.